src/main/scala/bitlap/sbt/analyzer/util/DependencyUtils.scala

```diff
@@ -124,51 +122,40 @@ object DependencyUtils {
     }
   }
 
-  private def isSelfArtifact(artifact: String, context: ModuleContext): Boolean = {
-    // processing cross-platform, module name is not artifact!
+  private def isSubModule(maybeModule: String, context: AnalyzerContext): Boolean = {
+    // Handles the cross-platform, module name is not equals to artifact!
     val currentModuleName =
-      context.ideaModuleIdSbtModuleNames.getOrElse(
+      context.moduleIdArtifactIdsCache.getOrElse(
         context.currentModuleId,
-        context.ideaModuleIdSbtModuleNames.getOrElse(
+        context.moduleIdArtifactIdsCache.getOrElse(
           Constants.SINGLE_SBT_MODULE,
-          context.ideaModuleIdSbtModuleNames.getOrElse(Constants.ROOT_SBT_MODULE, context.currentModuleId)
+          context.moduleIdArtifactIdsCache.getOrElse(Constants.ROOT_SBT_MODULE, context.currentModuleId)
         )
       )
 
     // NOTE: we don't determine the Scala version number.
-    if (context.isScalaNative) {
```

Comment:
Even on Native and Js platforms, isScalaNative and isScalaJs always return false.